WebsiteGreetz/src/test/java/WebsiteTest.java

```diff
@@ -24,56 +22,53 @@ private void logIn(WebDriver driver) throws InterruptedException {
         Thread.sleep(5000);
     }
 
-    private boolean findFavorite(WebDriver driver, int countOfFavorites) {
-        if (driver.findElements(By.className("center full-size")).size() > countOfFavorites)
-            return true;
+    private boolean findFavorite(int countOfFavorites) throws InterruptedException {
+        Thread.sleep(5000);
+        return driver.findElements(By.className("full-size")).size() > countOfFavorites;
+    }
 
-        return false;
+    private boolean find5and90euro() throws InterruptedException{
+        return driver.findElement(By.xpath("//div[@class='price-total' and contains(text(), '5,90')]")).isDisplayed();
     }
 
     @Test
     public void addAndCheckFavorite() throws InterruptedException {
         System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
         driver = new ChromeDriver();
         logIn(driver);
-
-        //ավելացնենք ֆավորիտ
-
         driver.get("https://www.greetz.nl/ballonnen/denken-aan");
         Thread.sleep(3000);
-
-        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']/.."));
-        WebElement favoriteItem = item.findElement(By.className("b-products-grid__item-action"));
+        WebElement item = driver.findElement(By.xpath("//a[@data-id='1142804226']"));
+        WebElement favoriteItem = item.findElement(By.xpath("//parent::div//child::a[@class='b-products-grid__item-action']"));
```

Comment:
actually this WebElement is not your favorite item as tells the variable name, but it is your `targetElementFavoriteButton`